utils/replace.py

```diff
@@ -0,0 +1,23 @@
+#python2
+
+import re
+import os
+from overviewer_core import configParser
+
+if __name__ == '__main__':
+    # Parse the overviewer's config file
+    my_parser = configParser.MultiWorldParser()
+    try:
+        my_parser.parse(os.path.expanduser('config.py'))
+    except configParser.MissingConfigException as e:
+        os.exit(-1)
+    config = my_parser.get_validated_config()
+
+    # Open the index.html in the output directory and add the key
+    with open(os.path.join(config['outputdir'], 'index.html'), 'r+') as fd:
```

Comment:
Do we really need to have `config['outputdir']` since in this Docker image, all renders are expected to go to `/home/minecraft/render/`? Removing that would remove the need for the `configParser`.

Reply:
When I created the pull request I was debating on if I should hardcode the value of the output directory or use the `configParser`. The tradeoff is a hardcoded value or an extra dependency.

Reply:
I'd agree with the value of having it configurable, except this Docker image expects the render to be placed to a specific path (perhaps we could make that more obvious?).